src/common/SteelSheets.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+#include "marlin_client.h"
+#include "eeprom.h"
+
+class SteelSheets {
+public:
+    static constexpr float Z_OFFSET_MIN = -2.0F;
+    static constexpr float Z_OFFSET_MAX = 0.0F;
```

Comment:
Suggestion:
Those are not macros so I would propose not to use ALL_UPPERCASE as identifier.